components/eventing-controller/api/v1alpha2/constants.go

```diff
@@ -20,4 +20,6 @@ const (
 	WebhookAuthClientSecret = "clientSecret"
 	WebhookAuthTokenURL     = "tokenUrl"
 	WebhookAuthScope        = "scope"
+
+	defaultMaxInFlightMessages = 10
```

Comment:
check if we still need it after changes. IMO, not a good idea to replicate configs, common configs could be in separate pkg so it can be reused in multiple places.

Reply:
agreed! removed!